tests/testthat/test-1-0-0rdlearn_lip.R

```diff
@@ -0,0 +1,26 @@
+test_that("learned cutoffs loop over lip(M)", {
+  # -------------------------------------------------------------------------- #
+  # esimate dif and lip
+  # -------------------------------------------------------------------------- #
+
+  set.seed(12345)
+  result <- rdlearn(y = "elig", x = "saber11", c = "cutoff",
```

Comment:
How long does it take for this test to run? If it takes too long, you can consider applying the function to a subset of the data, reduce the grid of `M` etc.